examples/SalesOrderApp/Testing.cs

```diff
@@ -42,5 +46,15 @@ public static string Json(object value)
         {
             return JsonConvert.SerializeObject(value, Formatting.Indented);
         }
+
+        public static async Task DisplayErrors(HttpResponseMessage response)
+        {
+            if (!response.IsSuccessStatusCode)
+            {
+                var responseString = await response.Content.ReadAsStringAsync();
+                JObject responseJson = JObject.Parse(responseString);
+                Console.WriteLine(responseJson.Last.Last);
```

Comment:
**MUST:** A JSON object is an unordered set of name/value pairs according to this [reference](https://stackoverflow.com/questions/16870416/does-the-sequence-of-the-values-matter-in-a-json-object/16870531#16870531). Please extract the text from the body property instead of the last parsed token.